### ISSUE TITLE
Need inventory api command

### ISSUE THREAD

**Issue by cheynearista** [aristanetworks/cvprac#62]
Need a generic get_inventory call so users can parse data for what they want. We provide get_device_by_name...but in some cases users would prefer to use MAC address or IP address.

**Comment by brokenpackets** [comment]
Could you elaborate on what you're looking for? We have a generic get_inventory that can search for a string value within the inventory.


```
    def get_inventory(self, start=0, end=0, query=''):
        ''' Returns the a dict of the net elements known to CVP.
            Args:
                start (int): The first inventory entry to return.  Default is 0
                end (int): The last inventory entry to return.  Default is 0
                    which means to return all inventory entries.  Can be a
                    large number to indicate the last inventory entry.
                query (string): A value that can be used as a match to filter
                    returned inventory list. For example get all switches that
                    are running a specific version of EOS.
        '''
        self.log.debug('get_inventory: called')
        data = self.clnt.get('/inventory/getInventory.do?'
                             'queryparam=%s&startIndex=%d&endIndex=%d' %
                             (urllib.quote_plus(query), start, end),
                             timeout=self.request_timeout)
        return data['netElementList']
```

**Comment by cheynearista** [comment]
@brokenpackets Right you are...my mistake. I made the assumption that if we had implemented get_inventory(),....get_device_by_name() would have called it directly. If the CVP API URI ever changes, we would need to change multiple lines of code instead of just one.

**Comment by brokenpackets** [comment]
I've got some free time today. Should be able to knock out changing get_device_by_name to call the get_inventory() function and pass the string as a param.

```
    def get_device_by_name(self, fqdn):
        ''' Returns the net element device dict for the devices fqdn name.
            Args:
                fqdn (str): Fully qualified domain name of the device.
            Returns:
                device (dict): The net element device dict for the device if
                    otherwise returns an empty hash.
        '''
        self.log.debug('get_device_from_name: fqdn: %s' % fqdn)
        data = self.clnt.get('/inventory/getInventory.do?'
                             'queryparam=%s&startIndex=0&endIndex=0'
                             % urllib.quote_plus(fqdn),
                             timeout=self.request_timeout)
        if len(data['netElementList']) > 0:
            for netelement in data['netElementList']:
                if netelement['fqdn'] == fqdn:
                    device = netelement
                    break
            else:
                device = {}
        else:
            device = {}
        return device
```

**Comment by mharista** [comment]
Fixed #63
